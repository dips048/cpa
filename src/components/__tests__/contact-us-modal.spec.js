import React from "react"
import renderer from "react-test-renderer"
import ContactUsModel from "../contact-us-modal"
import { render, fireEvent } from "@testing-library/react"

describe("ContactUsModel", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<ContactUsModel />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
