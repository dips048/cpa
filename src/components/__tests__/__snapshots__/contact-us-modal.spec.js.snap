// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactUsModel renders correctly 1`] = `null`;
